src/groupeddatatable/grouping.jl

```diff
@@ -201,7 +201,11 @@ function combine(ga::GroupApplied)
         idx[j + (1:n)] = gd.idx[start]
         j += n
     end
-    hcat!(gd.parent[idx, gd.cols], valscat)
+    if isa(valscat, DataTable)
```

Comment:
I don't really understand. `valscat = vcat(ga.vals)` and `ga.vals` is necessarily a `Vector{<:AbstractDataTable}`. So `valscat` is always an `AbstractDataTable`. Why do you need to call `append!` when it's not a `DataTable`?